sky/backends/backend_utils.py

```diff
@@ -1414,27 +1418,15 @@ def _get_tpu_vm_pod_ips(ray_config: Dict[str, Any],
     if len(stdout) == 0:
         logger.debug('No TPU VMs found with cluster name '
                      f'{cluster_name} in zone {zone}.')
-    if len(stdout.splitlines()) > 1:
+    all_json = json.loads(stdout)
+    if len(all_json) > 1:
         # Rare case, this could mean resource leakage. Hint user.
```

Comment:
I found this a bit confusing. This is normal for multi-node TPU VMs, right?

Reply:
Ah good point! I fixed it now so it'd only output when the number of rows != `expected_num_node`.